server/config/pipeline.js

```diff
@@ -56,15 +56,14 @@ function setUpPipelines(config, args, pathCascade) {
 
                 parseAnnotationRequires(pipeline, config, pathCascade, args)
                 pipeline.configOptions = mergeAdditionalAnnotationOptions(pipeline.configOptions, config, args);
-                // if any samples have been set (and therefore associated with barcodes) then we limit the run to those barcodes
-                if (config.run.samples.length) {
-                    if (pipeline.configOptions.limit_barcodes_to) {
-                        warn("Overriding your `limit_barcodes_to` options to those set via the barcode-sample mapping.")
-                    }
-                    pipeline.configOptions.limit_barcodes_to = [...getBarcodesInConfig(config)].join(',');
-                    verbose("config", `Limiting barcodes to: ${pipeline.configOptions.limit_barcodes_to}`)
+               // if any samples have been set (and therefore associated with barcodes) then we limit the run to those barcodes
+               if (config.run.samples.length) {
+                if (pipeline.configOptions.limit_barcodes_to) {
```

Comment:
why did the indentation here change? don't they have some prettier set up to automatically format files? :thinking:

Reply:
sorry, my mistake. I'll fix it.